config/initializers/content_domain.rb

```diff
@@ -7,3 +7,11 @@
                      else
                        "https://www.gov.uk"
                      end
+
+ORGANISATION_URL = if domain.include?  "dev"
+                     "http://whitehall-admin.dev.gov.uk"
+                   elsif domain.include? "preview"
+                     "http://www.preview.alphagov.co.uk"
+                   else
+                     "https://www.gov.uk"
+                   end
```

Comment:
So we're happy to remove this file and just call `Plek.current.website_root` from the helpers?